11ty/definitions/barbarian.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Barbarian
+slug: barbarian
+flag:
+  text: 'Colonial/Racist slur'
+  level: 'Avoid'
+defined: true
+speech: noun
+alt_words:
+  - cruel
+  - vicious
+  - obscene
+  - feral
+---
+someone who is "savagely cruel"; primitive ; unsophisticated
+
+
+#### Issues
+
+Barbarian originates from the greek word *bárbaros* meaning "babbler", to denote the "unintelligle sounds" (_"bar bar bar") made by foreign speakers.
+Similar words exist in many other languages, for the identical purpose labeling a "strange"/"foriegn" person/culture. 
+
+In modern day usage, you can notice this rhetoric being employed almost always for the purpose of demonizing "foreign" (more often than not from the global south) cultures - cultures that need to be civilized;
+cultures that need to be colonized to rescue them from themselves. A perfect lingual weapon for warmongers and racists alike. Equally repulsive sibling of "savage". 
```

Comment:
Let's break out "savage" as a "Related Words" section and link it.

cc: @ovlb Is there anything we should consider in doing this? I'd like for those words to eventually have some way of connecting.

Reply:
The one-way connection/link in the alternative words (e.g. barbarian -> savage) is automatic already. We can also add a «words that link here» section in the build. So I think there’s nothing special to consider currently. I will open an issue to discuss this further :)